src/popover/internal.tsx

```diff
@@ -190,9 +190,9 @@ function InternalPopover(
           {children}
         </span>
       )}
-      <FormFieldContext.Provider value={{}}>
+      <ResetContextsForModal>
```

Comment:
Why is it needed here?

Reply:
because popover content is not semantically "inline" with where it is rendered, so it shouldn't inherit any of these context values, same as a modal